acceptance/telemetry/timeout/test.toml

```diff
@@ -0,0 +1,9 @@
+[[Server]]
+Pattern = "POST /telemetry-ext"
+Response.Body = '''
+{
+    "errors": [],
+    "numProtoSuccess": 2
+}
+'''
+DelaySeconds = 5
```

Comment:
Could we add env var DATABRICKS_CLI_TELEMETRY_TIMEOUT and set it to 0.01 in tests so that we don't have to wait 3 seconds here?

Reply:
The tests run in parallel anyway so it does not add latency. 

Not configuring the timeout makes the test more robust, but I'm fine with making it configurable with an environment variable.

Reply:
Manually checked that running all tests with or without this one has similar runtimes.

Reply:
I'd prefer that all local tests run quickly. If I'm working on telemetry then I'm going to run the subset of tests in acceptance/telemetry and this will be on critical path.

Having DATABRICKS_CLI_TELEMETRY_TIMEOUT could be useful outside of tests as well.

Reply:
Added a env var, and configured timeout to 0.1 seconds.